src/Collision/collisionInterface.cpp

```diff
@@ -164,7 +164,7 @@ CollisionInterface::replaceContactSetWithPerimeter(ContactReport &contactSet)
 
 	coordT *array = new coordT[contactSet.size()*2];
 	coordT *ptr = &array[0];
-	int ptCount = 0;
+    volatile int ptCount = 0;
```

Comment:
from SO:
http://stackoverflow.com/questions/7996825/why-volatile-works-for-setjmp-longjmp

setjmp and longjmp clobber registers. If a variable is stored in a register, its value gets lost after a longjmp.

Conversely, if it's declared as volatile, then every time it gets written to, it gets stored back to memory, and every time it gets read from, it gets read back from memory every time. This hurts performance, because the compiler has to do more memory accesses instead of using a register, but it makes the variable's usage safe in the face of longjmping.

Another alternative solution from same SO article:
Well you could add some extra code to only make it volatile across the call to setjmp. Something like: int x; /* do stuff */ volatile int save_x = x; if(setjmp(buf)) { x = save_x; /* do stuff */ }. This maximizes performance before and after setjmp by using non-volatile variables, but ensures safety by using a volatile variable across the call.